helm/adelphi/templates/nosqlbench-jobs.yaml

```diff
@@ -26,24 +26,36 @@ spec:
         - name: ndots
           value: "1"
       volumeMounts:
+      - name: workspace
+        mountPath: /workspace
       - name: secret-volume
         mountPath: /secret
         readOnly: true
-      - name: workload
-        mountPath: /config/workload.yaml
-        subPath: workload.yaml
       - name: results-pv
         mountPath: /results
+    initContainers:
+    - name: extract-config
+      image: registry:5000/adelphi:0.2.0
```

Comment:
Original plan was to continue using the "pip install adephi + use adelphi" model seen in the Gemini example above before this change.  Had to shift away from this due to some dependency changes.

While working on this PR PyGithub released v1.55 which introduced a dependency on pynacl for supporting Github secrets.  pynacl in turn relies on native code to interact with libsodium, which meant the base image we used here needed to have gcc + some other dependencies installed.  That in turn led to the cassandra-driver install trying to compile it's uring extension now that it could do so... and suddenly the pip install of "adelphi" was taking ~20m due to all the added dependencies.

Rather than continue on with this path it seemed _much_ simpler to just create an image with "adelphi" already installed.  In addition to the _massive_ time savings this also has the effect of fixing the dependency versions used by "adelphi" _for usages within the workflow_.  These dependencies still float in setup.py (which we want) but we still get clear, fixed behaviour within Helm (which we also want).